circle.yml

```diff
@@ -3,17 +3,30 @@ general:
     ignore:
       - gh-pages # list of branches to ignore
 machine:
+  environment:
+    YARN_VERSION: 0.20.3
+    PATH: "${PATH}:${HOME}/.yarn/bin:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
   node:
-    version: 6.2.0
+    version: 4
```

Comment:
@cpojer, is DANGER_GITHUB_API_TOKEN a secret?
We can't run `danger` test without it and it won't be secure to pass DANGER_GITHUB_API_TOKEN to pull requests on Circle CI